bitcash/network/services.py

```diff
@@ -1,13 +1,13 @@
-import logging
 import os
 import requests
-from decimal import Decimal
 
-from bitcash.exceptions import InvalidNetwork
-from bitcash.network import currency_to_satoshi
-from bitcash.network.meta import Unspent
-from bitcash.network.transaction import Transaction, TxPart
+# Import supported endpoint APIs
+from bitcash.network.APIs.BitcoinDotComAPI import BitcoinDotComAPI
 
+# Dictionary of supported endpoint APIs
+ENDPOINT_ENV_VARIABLES = {"BITCOINCOM": BitcoinDotComAPI}
+
+# Default API call total time timeout
 DEFAULT_TIMEOUT = 30
 
 BCH_TO_SAT_MULTIPLIER = 100000000
```

Comment:
Same for `BCH_TO_SAT_MULTIPLIER`

Reply:
I see the `TODO` now. Maybe we can just do `from network.services import DEFAULT_TIMEOUT, BCH_TO_SAT_MULTIPLIER` for now.

Reply:
Can't be done because of circular imports